1.9/networking/service-and-pod-networking.md

```diff
@@ -17,6 +17,8 @@ DC/OS services and pods declare `networks` the same way. Three modes of networki
 
 In `host` networking, an application shares the network namespace of the Mesos agent process, typically the host network namespace.
 
+**Note:** The `Network.name` parameter cannot be used with `host` networking.
```

Comment:
so .. it's actually only useful for `container` mode. not `host` and not `container/bridge`.